docs/pages/release_notes.rst

```diff
@@ -1,14 +1,19 @@
-
 #############
 Release notes
 #############
 
 *******************
-Peanut (v6 alpha 1)
+Peanut (v6.0.0)
```

Comment:
we should make more clear this version isn't released yet, so maybe omit the version/reprhase it?

Reply:
L9 
> **Release date:** TBD